docs/csharp/cs-dev-kit-faq.md

```diff
@@ -215,6 +215,12 @@ If you are encountering issues with Test Explorer, you can enable diagnostic log
 
 This information will help in diagnosing and resolving the issue more efficiently.
 
+### Known Issues
+
+- Using xUnit v3 with package Microsoft.Testing.Platform on macOS can cause test discovery and debugging issues in the Test Explorer for C# Devkit and Visual Studio.
+Workaround: Downgraded to xUnit v2 or use VSTest instead.
```

Comment:
Can we link to an existing issue to help users follow along and/or share additional feedback?